triton/examples/openpose/openpose.rst

```diff
@@ -4,20 +4,26 @@ OpenPose
 This uses :doc:`Singularity containers </triton/usage/singularity>`,
 so you should refer to that page first for general information.
 
-OpenPose has been compiled against Atlas and Caffe, CUDA and cuDNN.
-Image is based on a `bvlc/caffe:gpu <https://hub.docker.com/r/bvlc/caffe>`_ base image.
+OpenPose has been compiled against OpenBlas, Caffe, CUDA and cuDNN.
+Image is based on a `nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04 <https://hub.docker.com/r/nvidia/cuda/tags>`_ docker image.
 
 Dockerfile for this image is available
-`here <https://github.com/AaltoScienceIT/scienceit-dockerfiles/blob/master/OpenPose/Dockerfile>`_.
+`here <https://raw.githubusercontent.com/AaltoScienceIT/scienceit-dockerfiles/master/OpenPose/v1.5.1/Dockerfile>`_.
 
-Within the container OpenPose is installed under ``/opt/openpose-master``. Due to
-the way the examples are organized, the ``singularity_wrapper`` changes the
-working directory to ``/opt/openpose-master``.
+Within the container OpenPose is installed under ``/opt/openpose``. Due to
+the way the libraries are organized, ``singularity_wrapper`` changes the
+working directory to ``/opt/openpose``.
 
-Example :download:`sbatch script </triton/examples/openpose/openpose.slrm>` is
-shown below.
+Running OpenPose example
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+One can run this example with ``srun``::
 
-Usage
-~~~~~
+  wget https://raw.githubusercontent.com/AaltoScienceIT/scicomp-docs/master/triton/examples/openpose/openpose.sh
```

Comment:
I like the way wget is used here, I hadn't thought of that.  But also teaching someone to open the editor and copy/paste is what they'll need to do next anyway...